locales/en-US/app.ftl

```diff
@@ -61,7 +61,7 @@ emailOptInInput =
 emailOptInButton = Sign me up
 emailOptInSkip = Skip
 emailOptInConfirmationTitle = Email Sent
-emailOptInSuccessMessage = When you get a chance, head over to your inbox to verify your email address. Thank you!
```

Comment:
Yeah, we should. I think the general rule of thumb would be to change the ID whenever the meaning of the string changes.